src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -904,6 +905,19 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    void AppLogic::_ApplyLanguageSettingChange()
+    {
+        using ApplicationLanguages = winrt::Windows::Globalization::ApplicationLanguages;
+
+        const auto language = _settings.GlobalSettings().Language();
+        const auto primaryLanguageOverride = ApplicationLanguages::PrimaryLanguageOverride();
+
+        if (primaryLanguageOverride != language)
+        {
+            ApplicationLanguages::PrimaryLanguageOverride(language);

```

Comment:
Just to confirm- this works fine if you call it with an empty string? Very cool.